bin/acm

```diff
@@ -5,23 +5,93 @@ require 'thor'
 
 class AkamaiCloudletManagerCli < Thor
 
-  desc "clone_version", "Clones the current version"
+  desc "get_policy_version", "Get policy versions"
   long_desc <<-LONGDESC
-  Clones the current activated version. All new updates are performed in cloned version
+  Get the policy versions.
+    With --policy_id, policy on which version to be cloned
     LONGDESC
-  def clone_version
-    puts "clones the version"
+  option :policy_id, required: true
+  def get_policy_version
+    puts "Policy version:"
+    policy_version = AkamaiCloudletManager::PolicyVersion.new({policy_id: options[:policy_id] })
+    puts policy_version.versions
   end
 
-  desc "update_version", "Updates the version"
+  desc "clone_policy_version", "Clones the current policy version"
   long_desc <<-LONGDESC
-  Updates the version
-    With --version, version which needs to be updated
+  Clones the current activated version of a policy. All new updates are performed in cloned version
+    With --policy_id, policy on which version to be cloned
+    LONGDESC
+  option :policy_id, required: true
+  def clone_policy_version
+    puts "clones the policy version"
+    policy_version = AkamaiCloudletManager::PolicyVersion.new({policy_id: options[:policy_id] })
+    begin
+      latest_version = JSON.parse(policy_version.versions).first["version"].to_s
+      policy_version.create(latest_version)
+    rescue => err
+      puts "cloning failed!"
+      puts "Exception: #{err}"
+      err
+    end
+  end
+
+  desc "update_policy_version", "Updates the policy version"
+  long_desc <<-LONGDESC
+  Updates the policy version. A policy version can be updated only in draft mode not when it is activated.
+    With --policy, policy on which version to be updated
+    With --draft_version, version which needs to be updated, this must be inactive version
+    With --file_path, file_path from which rules can be constructed and updated to policy version
+
+    LONGDESC
+
+  option :policy_id,  required: true
```

Comment:
For `option` looks like we have [method_options](https://github.com/erikhuda/thor/wiki/Method-Options) now

Reply:
👍